typed-racket-lib/typed-racket/typecheck/internal-forms.rkt

```diff
@@ -157,6 +155,8 @@
     (assert-predicate-internal type predicate)]
   [type-declaration
     (:-internal id:identifier type)]
+  [unsafe-type-declaration
+    (unsafe-:-internal id:identifier type)]
   ;; the check field indicates whether this signature is being 
```

Comment:
`unsafe-:-internal` us too cute a name for inside TR's implementation, IMHO. I think it should be less punny on `:`.

Maybe `unsafe-declare-type` or similar?

Reply:
I was following the convention in the previous line. If we decide to change that convention, it shouldn't be part of this pull request.

Reply:
Aha, I missed that -- sorry.